libs/documentation/src/lib/storybook/autocomplete/autocomplete.stories.ts

```diff
@@ -389,6 +391,26 @@ FreeText.parameters = {
   stackblitzLink: generateStackblitzLink('autocomplete', 'free-text'),
 };
 
+export const CheckBox: Story = (args) => ({
+  template: `<sds-autocomplete-checkbox-demo></sds-autocomplete-checkbox-demo>`,
+  props: {
+    ...args,
+  },
+});
+CheckBox.parameters = {
+  controls: {
+    disabled: true,
+    hideNoControlsWarning: true,
+  },
+  actions: { disabled: true },
+  preview: generateConfig(
+    'storybook/autocomplete/autocomplete-checkbox',
+    'AutocompleteCheckboxModule',
+    'sds-autocomplete-checkbox-demo'
+  ),
+  stackblitzLink: generateStackblitzLink('autocomplete', 'free-text'),
```

Comment:
Update this so that the demo is generated correctly.

Reply:
done